src/attr/_make.py

```diff
@@ -1210,9 +1214,9 @@ def _add_cmp(cls, attrs=None):
 
 def _make_repr(attrs, ns):
     """
-    Make a repr method for *attr_names* adding *ns* to the full name.
+    Make a repr method that includes relevant *attrs*, adding *ns* to the full
```

Comment:
Not sure if y'all do this style-wise, but this breaks the docstring summary line now that it's 2 lines.

Reply:
there are may more with multi-line ‘summary’ docstring lines. and this is internal api only so not really an issue at all (not used in docs or processed by external tools).